app/assets/javascripts/admin/bulk_order_management.js.coffee

```diff
@@ -162,6 +163,12 @@ angular.module("ofn.admin").controller "AdminOrderMgmtCtrl", [
       $scope.supplierFilter = $scope.suppliers[0].id
       $scope.orderCycleFilter = $scope.orderCycles[0].id
       $scope.quickSearch = ""
+
+    $scope.weightAdjustedPrice = (lineItem, oldValue) ->
+      if lineItem.unit_value <= 0
+        lineItem.unit_value = lineItem.units_variant.unit_value
+      lineItem.price = lineItem.price * lineItem.unit_value / oldValue
```

Comment:
I think this really needs to be able to handle the zero unit_values. At the moment it just seems to crash every time I try to change the unit_value of a line_item from 0 to 1, because it is trying to determine the new price by dividing lineItem.unit_value by oldValue, which is zero.